internal/binutils/binutils.go

```diff
@@ -140,7 +144,76 @@ func initTools(b *binrep, config string) {
 		b.addr2line, b.addr2lineFound = findExe("gaddr2line", append(paths["addr2line"], defaultPath...))
 	}
 	b.nm, b.nmFound = findExe("nm", append(paths["nm"], defaultPath...))
-	b.objdump, b.objdumpFound = findExe("objdump", append(paths["objdump"], defaultPath...))
+	b.objdump, b.isLLVMObjdump, b.objdumpFound = findObjdump(append(paths["objdump"], defaultPath...))
+}
+
+// findObjdump finds and returns path to preferred objdump binary.
+// Order of preference is: llvm-objdump, objdump.
+// On MacOS only, also looks for gobjdump with least preference.
+// Accepts a list of paths and returns:
+// a string with path to the preferred objdump binary if found,
+// or an empty string if not found;
+// a boolen indicating if it is an LLVM objdump;
+// a boolean if any acceptable objdump was found.
+func findObjdump(paths []string) (string, bool, bool) {
+	objdumpNames := []string{"llvm-objdump", "objdump"}
+	if runtime.GOOS == "darwin" {
+		objdumpNames = append(objdumpNames, "gobjdump")
+	}
+
+	for _, objdumpName := range objdumpNames {
+		if objdump, objdumpFound := findExe(objdumpName, paths); objdumpFound {
+			cmdOut, err := exec.Command(objdump, "--version").Output()
+			if err != nil {
+				continue
+			}
+			if llvmObjdumpFound := isLLVMObjdump(string(cmdOut)); llvmObjdumpFound {
+				return objdump, llvmObjdumpFound, llvmObjdumpFound
```

Comment:
Done. 
Is this a style preference or is there a specific reason? 
Since there are two bools, specifying the variable makes it more readable I think.

Reply:
If a variable is returned, I'd assume its value can be different at this point. Understanding that its value is actually the same is additional mental load which is nice to avoid when reading code.